enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/DocumentVerificationRepository.java

```diff
@@ -47,16 +48,10 @@ public interface DocumentVerificationRepository extends JpaRepository<DocumentVe
             "AND d.status IN (:statuses)")
     int failVerifications(String activationId, Date timestamp, List<DocumentStatus> statuses);
 
-    @Modifying
-    @Query("UPDATE DocumentVerificationEntity d " +
-            "SET d.status = com.wultra.app.enrollmentserver.model.enumeration.DocumentStatus.FAILED, " +
-            "    d.usedForVerification = false, " +
-            "    d.errorDetail = :errorMessage, " +
-            "    d.errorOrigin = :errorOrigin, " +
-            "    d.timestampLastUpdated = :timestamp " +
+    @Query("SELECT d.id FROM DocumentVerificationEntity d " +
             "WHERE d.timestampLastUpdated < :cleanupDate " +
             "AND d.status IN (:statuses)")
```

Comment:
Are the brackets required in `(:statuses)`?

Reply:
No, it works without them.